plutus-benchmark/validation/Main.hs

```diff
@@ -38,7 +39,7 @@ loadPlcSource file = do
      Right p                    -> return $ () <$ p
 
 benchCek :: Term () -> Benchmarkable
-benchCek program = nf (UPLC.unsafeEvaluateCek PLC.defBuiltinsRuntime) program
+benchCek program = nf (UPLC.unsafeEvaluateCek defaultCekCosts PLC.defBuiltinsRuntime) program
```

Comment:
We should probably export this from the `PlutusCore` module too, then this could also be `PLC.defaultCekCosts` rathe than importing `ExBudgetingDefaults` directly.

Reply:
Good idea.  That's much better.